pom.xml

```diff
@@ -31,7 +31,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.build.java.version>1.8</project.build.java.version>
-        <elasticsearch.version>[7.4.0,)</elasticsearch.version>
+        <elasticsearch.version>7.4.0</elasticsearch.version>
```

Comment:
We are using ES's version for the plugin version, so can you replace `7.0.4` with `${project.version}`?
To build the plugin with specific version, we only need to modify this line <version>7.0.4</version.

Reply:
Changed. Please review again.